bigtable/src/main/java/com/m/examples/bigtable/Quickstart.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.m.examples.bigtable;
+
+// START [bigtable_quickstart_veneer]
+import com.google.api.gax.rpc.NotFoundException;
+import com.google.cloud.bigtable.data.v2.BigtableDataClient;
+import com.google.cloud.bigtable.data.v2.BigtableDataSettings;
+import com.google.cloud.bigtable.data.v2.models.Row;
+import com.google.cloud.bigtable.data.v2.models.RowCell;
+
+public class Quickstart {
+  public static void quickstart(String projectId, String instanceId, String tableId)
+      throws Exception {
```

Comment:
Don't throw exceptions from this function. Instead if you have to, you can catch the exception and log it. See this [example](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#exception-handling).